src/components/scenes/CreateWalletSelectCryptoScene.tsx

```diff
@@ -1,144 +1,255 @@
-import { EdgeAccount } from 'edge-core-js'
 import * as React from 'react'
-import { Alert, FlatList, View } from 'react-native'
-import IonIcon from 'react-native-vector-icons/Ionicons'
+import { FlatList, Switch, View } from 'react-native'
+import { sprintf } from 'sprintf-js'
 
-import { getSpecialCurrencyInfo } from '../../constants/WalletAndCurrencyConstants'
+import { enableTokensAcrossWallets, MainWalletCreateItem, PLACEHOLDER_WALLET_ID, splitCreateWalletItems } from '../../actions/CreateWalletActions'
+import { useHandler } from '../../hooks/useHandler'
+import { useWatch } from '../../hooks/useWatch'
 import s from '../../locales/strings'
-import { connect } from '../../types/reactRedux'
+import { useDispatch, useSelector } from '../../types/reactRedux'
 import { NavigationProp } from '../../types/routerTypes'
-import { CreateWalletType, FlatListItem } from '../../types/types'
-import { getCreateWalletTypes } from '../../util/CurrencyInfoHelpers'
 import { SceneWrapper } from '../common/SceneWrapper'
-import { cacheStyles, Theme, ThemeProps, withTheme } from '../services/ThemeContext'
+import { ListModal } from '../modals/ListModal'
+import { Airship, showError } from '../services/AirshipInstance'
+import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
 import { CreateWalletSelectCryptoRow } from '../themed/CreateWalletSelectCryptoRow'
+import { EdgeText } from '../themed/EdgeText'
+import { Fade } from '../themed/Fade'
+import { MainButton } from '../themed/MainButton'
 import { OutlinedTextInput } from '../themed/OutlinedTextInput'
 import { SceneHeader } from '../themed/SceneHeader'
+import { filterWalletCreateItemListBySearchText, getCreateWalletList, WalletCreateItem } from '../themed/WalletList'
+import { WalletListCurrencyRow } from '../themed/WalletListCurrencyRow'
 
 type OwnProps = {
-  navigation: NavigationProp<'createWalletReview'>
+  navigation: NavigationProp<'createWalletSelectCrypto'>
 }
-type StateProps = {
-  account: EdgeAccount
-}
-type Props = StateProps & OwnProps & ThemeProps
 
-type State = {
-  selectedWalletType: string
-  searchTerm: string
-}
+type Props = OwnProps
```

Comment:
Same thing here. Just call it `Props` and don't bother with the `OwnProps` and the alias.